core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1278,6 +1278,30 @@ public String toString() {
               + "used when Zookeeper is enabled, since Zookeeper already stores the master "
               + "addresses.")
           .build();
+  public static final PropertyKey MASTER_FILE_ACCESS_TIME_JOURNAL_FLUSH_INTERVAL =
+      new Builder(Name.MASTER_FILE_ACCESS_TIME_JOURNAL_FLUSH_INTERVAL)
+          .setDefaultValue("1h")
+          .setDescription("The minimum interval between files access time update journal entries "
+              + "get flushed. Setting it to a non-positive value will make the the journal update "
+              + "synchronous.")
```

Comment:
In the description, can you add the implications of synchronous flushes (always consistent but slows down reads), as well as an async flush interval (does not affect reads, but some accesses may be lost on crashes)?

Reply:
Done.